validate_sptx/validate_sptx.py

```diff
@@ -48,7 +48,10 @@ def validate(experiment_json: str, fuzz: bool=False) -> bool:
 
     # validate experiment.json
     experiment_validator = SpaceTxValidator(_get_absolute_schema_path('experiment.json'))
-    valid &= experiment_validator.validate_object(experiment, name, fuzz=fuzz)
+    if fuzz:
```

Comment:
why not a top-level command fuzz()?

Reply:
My instinct is that a separate command makes a lot more sense.  However, your concerns about code duplication is on point.

In the long term, I'd like the validation schema to somehow reference the child schema docs so this traversal can be done programmatically rather than explicitly.  e.g., the schema for the `codebook` field in experiment.json should reference the codebook schema.  As we are validating the `codebook` field, it should be know to load up `codebook/codebook.json` and validate the referenced file against that.

My guess is that jsonschema does not provide facilities to do this, and we would need to build our own internal referencing system.

Let's leave the code as is for now, and return to it when we have cycles.

Reply:
Interesting. I actually remembered the parent schema's referring to the child schemas. Perhaps this was something on the HCA side. I've listed this as a follow-up in https://github.com/spacetx/starfish/issues/539.

On the code side, another idea (also for more cycles) would be to convert this to an instance as opposed to just a function. That might allow for both clarity and code-reuse.

Reply:
e.g. I think `codebook` is missing its `$ref`: https://github.com/spacetx/starfish/blob/master/validate_sptx/schema/experiment.json#L14

Reply:
> Interesting. I actually remembered the parent schema's referring to the child schemas. Perhaps this was something on the HCA side. I've listed this as a follow-up in #539.

My current level of understanding is that explicit `$ref` linking means that the file expects to see an `object` in place of the schema, rather than a file containing the object, meaning that we'd end up with one big json blob. If that's _wrong_, then there is a big opportunity for us to clean up the specification by removing these implicit links...

Reply:
No, this is the same issue as https://github.com/spacetx/starfish/pull/647#discussion_r220971186, i.e. you can pull other schema files into your main schema, but I don't believe you can currently pull json instance documents into the current json document.

Reply:
Some glimmer of hope in json-refs, e.g. https://jsonref.readthedocs.io/en/latest/